src/node_modules/app/elasticsearch/config.js

```diff
@@ -314,3 +314,77 @@ export const AGG_DOC_URLS = {
 	significant_text: 'https://www.elastic.co/guide/en/elasticsearch/reference/6.4/search-aggregations-bucket-significanttext-aggregation.html',
 	terms: 'https://www.elastic.co/guide/en/elasticsearch/reference/6.4/search-aggregations-bucket-terms-aggregation.html',
 };
+
+export const defaultValues = {
+	cardinality: {
```

Comment:
I'd sort this object (and its nested objects) by key to ease reading & visual search